spellbook/models/safe/ethereum/safe_ethereum_eth_transfers.sql

```diff
@@ -0,0 +1,40 @@
+{{ 
+    config(
+        materialized='incremental',
+        alias='eth_transfers',
+        unique_key = 'address',
```

Comment:
this table is using merge, however, it appears address is not unique. i ran this the first time, loaded as expected, then the second time it failed on unique constraints. you may need to either add multiple fields and/or add a column which concats a few for unique value

Reply:
True, my bad. I added `tx_hash` field that should be always unique.